index.html

```diff
@@ -10,7 +10,379 @@
+                            <img class="main__team-socials-img" src="./images/facebook.svg" alt="twitter-icon" />
+                        </div>
+                    </div>
+                </div>
+                <div class="main__team-member-card">
+                    <div class="main__team-container-img">
+                        <img class="main__team-img" src="./images/cover (2).png" alt="employee" />
+                    </div>
+                    <div class="main__team-info">
+                        <div class="main__team-name">Avie Beaton</div>
+                        <div class="main__team-job">CO Founder</div>
+                        <div class="main__team-socials">
+                            <img class="main__team-socials-img" src="./images/instragram.svg" alt="facebook-icon" />
+                            <img class="main__team-socials-img" src="./images/twitter.svg" alt="instragram-icon" />
+                            <img class="main__team-socials-img" src="./images/facebook.svg" alt="twitter-icon" />
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+
+        <div class="main__contact">
+            <div class="main__contact-contents">
+                <div class="main__contact-sub-title">Contact us</div>
+                <div class="main__contact-title">Make an appointment</div>
+                <div class="main__contact-input-container">
+                    <div class="main__contact-input-container1">
+                        <input class="main__contact-name-input" type="text" placeholder="Full Name" />
+                        <input class="main__contact-email-input" type="text" placeholder="example@gmail.com" />
+                    </div>
+                    <div class="main__contact-layout-input-container1">
+                        <select class="contact-layout__input">
+                            <option value="">Please Select</option>
+                            <option value=""></option>
+                        </select>
+                        <select class="contact-layout__input">
+                            <option value="en">4:00 Available</option>
+                            <option value=""></option>
+                        </select>
+                    </div>
+                    <div class="main__contact-layout-message">
+                        <textarea class="main__contact-layout-input-messages" name="" id="" cols="30" rows="10">
+                         Message 
+                        </textarea>
+                    </div>
+                    <button class="main__contact-booking-button main__contact-booking-button--link">Book an
+                        appointment</button>
+                </div>
+            </div>
+            <div class="main__contact-agency">
+                <div class="main-contact__agency-content">
```

Comment:
Same issue here you don't have a BLOCK called `main-contact` but you are using it here.